### PR TITLE
polyhedral geometry: added constructions delpezzo and fano_simplex

### PR DESCRIPTION
**Pull request by micjoswig**
@lkastner Please check.  If you like it, please merge.  If you already did it, you can also kill this.

### CODE REVIEW

File: src/PolyhedralGeometry/Polyhedron/standard_constructions.jl [hunk 1]
```diff
@@ -809,3 +809,44 @@ julia> volume(p)
 ```
 """
 gelfand_tsetlin(lambda::AbstractVector) = Polyhedron{fmpq}(Polymake.polytope.gelfand_tsetlin(Polymake.Vector{Polymake.Rational}(lambda), projected = false))
+
+@doc Markdown.doc"""
+
```
Comment (fingolfin) [inline src/PolyhedralGeometry/Polyhedron/standard_constructions.jl]:
```suggestion
```

File: src/PolyhedralGeometry/Polyhedron/standard_constructions.jl [hunk 2]
```diff
@@ -809,3 +809,44 @@ julia> volume(p)
...
+A normal toric variety
+
+julia> issmooth(X)
+true
+```
+"""
+fano_simplex(d::Int) = Polyhedron{fmpq}(Polymake.polytope.fano_simplex(d))
+
+@doc Markdown.doc"""
+
```
Comment (fingolfin) [inline src/PolyhedralGeometry/Polyhedron/standard_constructions.jl]:
```suggestion
```

**Review by fingolfin — APPROVED** [review]
Looks good to me. I'll happily let @lkastner have the final word, though!

The extra newlines are unimportant and appear in other docstrings in that file and can be removed everywhere in a future PR.

**Review by lkastner — APPROVED** [review]
I will merge once the tests have finished running.

**Pull request merged**